cpp/src/arrow/c/bridge_test.cc

```diff
@@ -3694,7 +3695,10 @@ TEST_F(TestSchemaRoundtrip, Dictionary) {
 }
 
 TEST_F(TestSchemaRoundtrip, UnregisteredExtension) {
-  TestWithTypeFactory(uuid, []() { return fixed_size_binary(16); });
+  TestWithTypeFactory(complex128, []() {
```

Comment:
It looks like you're removing uuid from a number of tests here.  Why is that?

Reply:
Some of these were checking registering / deregistering mechanics for extension types and it wasn't practical to use UUID type for that so I either removed them or replaced with something equivalent. These tests typically assumed uuid was not globally available extension at the start.